app/main.py

```diff
@@ -21,15 +23,19 @@ def serve_cars(self, cars: list[Car]) -> float:
             self.wash_single_car(i)
         return income
 
```

Comment:
It would be better to use "car" instead of "i" in for-loop

Reply:
i is a bad name for variable, use more descriptive names for variables